### ISSUE TITLE
Flag to bfstrip/bfmake to preserve unknown characters

### ISSUE THREAD

**Issue by ColonelThirtyTwo** [kspalaiologos/asmbf#124]
I'm working with a brainfuck-superset language that adds a few characters that I would like to use. `bfstrip` though appears to filter those characters out. Can there be a flag to `bfstrip` (and `bfmake`) to preserve those characters?

Example below. `raw` instructions are the extension symbols.

```
col@col-laptop-asus ~> cat test.asm

mov r1, 1
nav r1
raw .}
mov r1, 2
nav r1
raw .)
mov r1, 20
nav r1
raw .'
raw .@
col@col-laptop-asus ~> bfmake ./test.asm
col@col-laptop-asus ~> cat ./test.b
+>+[<[>-]>[>]<<[>>+<<-]>>[-<<+>>>>>[-]+[-]++[-]++++++++++++++++++++<<<]<<[>>+<<-]>>[-<<+>>]<]⏎
col@col-laptop-asus ~> bfmake -s ./test.asm
col@col-laptop-asus ~> cat ./test.b
+>+[<[>-]>[>]<<[>>+<<-]>>[-<<+>>>>>[-]+}[-]++)[-]++++++++++++++++++++'@<<<]<<[>>+<<-]>>[-<<+>>]<]⏎
col@col-laptop-asus ~>
```

(Specifically, the "extended brainfuck" is the script for the ChemiCompiler from the game Space Station 13 - it adds a few symbols to interact with the game world)

**Comment by kspalaiologos** [comment]
Use `bfmake -s`

**Comment by ColonelThirtyTwo** [comment]
Yeah, I'm aware of the `-s` flag, but that also means foregoing the optimizations that `bfstrip` does, hence this ticket.

**Comment by kspalaiologos** [comment]
resolved in 4028e37 and 1f462af - use `bfmake -q` to build your project.
